<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Geo Web Services</string>
  </property>
  <widget class="QLineEdit" name="WSName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listWS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>154</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateNewWS">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Update/New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteWS">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>77</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="WSUrl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>320</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="VariablesList">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="insertVar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Insert variable</string>
   </property>
  </widget>
  <widget class="QLabel" name="helper">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
